packages/api/src/APIClient.ts

```diff
@@ -795,6 +799,81 @@ export default class APIClient {
 				'Delete Client'
 			),
 	}
+	public readonly bills = {
+		list: (accountId: string, queryBuilders?: QueryBuilderType[]): Promise<Result<{ bills: Bills[] }>> =>
+			this.call(
+				'GET',
+				`accounting/account/${accountId}/bills/bills${joinQueries(queryBuilders)}`,
+				{ transformResponse: transformBillsListResponse },
+				null,
+				'List Bills'
+			),
+		single: (accountId: string, billId: number): Promise<Result<{ bill: Bills }>> =>
+			this.call(
+				'GET',
+				`accounting/account/${accountId}/bills/bills/${billId}`,
+				{ transformResponse: transformBillsResponse },
+				null,
+				'Get Bill'
+			),
+		create: (bill: Bills, accountId: string): Promise<Result<Bills>> =>
+			this.call(
+				'POST',
+				`accounting/account/${accountId}/bills/bills`,
+				{
+					transformResponse: transformBillsResponse,
+					transformRequest: transformBillsRequest,
+				},
+				bill,
+				'Create Bill'
+			),
+		delete: (accountId: string, billId: number): Promise<Result<Bills>> =>
+			this.call(
+				'PUT',
+				`accounting/account/${accountId}/bills/bills/${billId}`,
+				{
+					transformResponse: transformBillsResponse,
+					transformRequest: transformBillsRequest,
+				},
+				{ visState: 1 },
+				'Delete Bill'
+			),
+		archive: (accountId: string, billId: number): Promise<Result<Bills>> =>
+			this.call(
+				'PUT',
+				`accounting/account/${accountId}/bills/bills/${billId}`,
+				{
+					transformResponse: transformBillsResponse,
+					transformRequest: transformBillsRequest,
+				},
+				{ visState: 2 },
+				'Archive Bill'
+			),
+	}
+	public readonly billPayments = {
+		create: (billPayment: BillPayments, accountId: string): Promise<Result<BillPayments>> =>
```

Comment:
Looks like they're missing from our API docs, but Bill Payments should also have a `list`, `single`, and `delete`.
I can add examples to the docs.